app/Platform/routes/platform-routes.php

```diff
@@ -1,10 +1,24 @@
 <?php
 
 use App\Platform\Http\Controllers\MemberSocialAuthController;
+use App\Platform\Http\Controllers\RegisterMemberController;
+use Illuminate\Foundation\Auth\EmailVerificationRequest;
 use Illuminate\Support\Facades\Route;
 
-Route::group(['middleware' => ['auth:sanctum', 'treblle', 'stateful']], function () {
+Route::group(['middleware' => ['auth:api', 'treblle']], function () {
+
+    Route::post('/register', [RegisterMemberController::class, 'store'])->withoutMiddleware(['auth:api']);
+
+    Route::get('/register/{provider}/redirect', [MemberSocialAuthController::class, 'redirect'])->withoutMiddleware(['auth:api']);
+    Route::get('/register/{provider}/callback', [MemberSocialAuthController::class, 'handleCallback'])->withoutMiddleware(['auth:api']);
 
-    Route::get('/register/{provider}/redirect', [MemberSocialAuthController::class, 'redirect'])->withoutMiddleware(['auth:sanctum']);
-    Route::get('/register/{provider}/callback', [MemberSocialAuthController::class, 'handleCallback'])->withoutMiddleware(['auth:sanctum']);
 });
+
+Route::get('/email/verify/{id}/{hash}', function (EmailVerificationRequest $request) {
+    $request->fulfill();
+
+    return response()->json([
```

Comment:
Please create an EmailVerificationController and move this code there.

Also, don't forget to translate the message to Macedonian using language files.

Reply:
And also, don't forget to add API docs for this endpoint.